adoc/MAIN.release-notes.adoc

```diff
@@ -20,6 +22,42 @@ Release notes usually only list changes that happened between two subsequent rel
 
 {productname} is an enterprise-ready {kube}-based container management solution used by application development and DevOps teams to more easily and efficiently deploy and manage containerized applications and services. Enterprises use {productname} to reduce application delivery cycle times and improve business agility.
 
+== Changes in 4.0.1
+
+* Updated Gangway container image (see <<required-actions-401>>)
+* Added link:https://documentation.suse.com/suse-caasp/4/single-html/caasp-deployment/#_airgapped_deployment[air gap deployment instructions]
+* Various bug fixes and improvements
```

Comment:
Can we please get a more detailed list of "Various bug fixes and improvements" ?
See for example SUSE Manager [changes](https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.0/#_version_4_0_2) and [patches](https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.0/#_version_4_0_2_2) for 4.0.2

/cc @jordimassaguerpla

Reply:
> Can we please get a more detailed list of "Various bug fixes and improvements" ?
> See for example SUSE Manager [changes](https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.0/#_version_4_0_2) and [patches](https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.0/#_version_4_0_2_2) for 4.0.2
> 
> /cc @jordimassaguerpla

There is nothing more to add apart from the changed gangway image according to jordi. The rest of the time between 4.0.0 and this was spent on tech debt and training. I added this line to avoid adding the list of open and closed bugzilla bugs because that is no use to anyone unfamiliar with the issues.

Is it enough to remove this line? We will add more details in the next release notes when there is more stuff to mention.

Reply:
4.0.1 is only about the gangway fix plus airgap instructions.

We did not put a list of bsc fixes (in this case would be only one) because this info is already in the patchinfo.

@kkaempf What is your expectation here? Would you like to see in the release notes the bsc numbers that were fixed? Or just the relevant information, like new features or installing instructions as we've done?

Reply:
> We did not put a list of bsc fixes (in this case would be only one) because this info is already in the patchinfo.

I think the patchinfo (just the bugzilla entries with their one-line summary) should be in the release notes. Adding it is easily scriptable (and I volunteer to help :wink:)
> 
> @kkaempf What is your expectation here? Would you like to see in the release notes the bsc numbers that were fixed?

Yes

> Or just the relevant information, like new features or installing instructions as we've done?

We need both, a high level overview of (user-)visible changes (like new features, changed behaviour, version updates of upstream components, ...) and a list of bugzilla entries.


It's fine as is for the 4.0.1 update but we should add more detail to the 4.0.2 release notes.

Reply:
ok. Let's move forward then and do it better for 4.0.2. Thanks.

Reply:
Let's give the customer a _single_ place to obtain all the information he needs to know what the MU does so he can decide on the needs and risks of applying this MU.

Spreading the information across release notes and patchinfo is not practical in this regard.

Reply:
> Let's give the customer a _single_ place to obtain all the information he needs to know what the MU does so he can decide on the needs and risks of applying this MU.
> 
> Spreading the information across release notes and patchinfo is not practical in this regard.

So SUMA takes the info from the "patchinfo" that goes onto patchfinder and OBS and puts them in the release notes as well right? So are they still writing the old patchinfo?

Reply:
> So SUMA takes the info from the "patchinfo" that goes onto patchfinder and OBS and puts t
them in the release notes as well right?

Yes

> So are they still writing the old patchinfo

doesn't matter - the information contained in the patchinfo matters

Reply:
> > So are they still writing the old patchinfo
> 
> doesn't matter - the information contained in the patchinfo matters

I'm just trying to find out if we need to stuff _on top_ of the existing work or if we can shift some existing effort into doing this better. From what I understand patchinfo is used by the QAM people and customers to review, since those patches don't really apply to CaaSP as much as they do to SLES or others we could _stop_ writing the patchinfo file and _instead_ add the change information direclty to the release notes. But I don't know how much Stefan Barth will like that or if I should care ...

Reply:
"since those patches don't really apply to CaaSP as much as they do to SLES " - No. We have an agreement with maintenance that the 'same rules' apply to CaaSP, be it RPM- or container-based updates. Both must be accompanied with more detailed information about _why_ there's an update - basically a patchinfo.

Reply:
" we could stop writing the patchinfo file and instead add the change information direclty to the release notes. " - no again since customers are requesting this information in an easily accessible place. While [patch finder](https://download.suse.com/patch/finder) will be decommissioned soon, the information was considered important/valuable enough to move the patch finder functionality under scc.suse.com.